packages/paste-core/components/badge/src/utils.ts

```diff
@@ -1,66 +1,11 @@
-import type {TextColorOptions, BackgroundColorOptions} from '@twilio-paste/style-props';
+import {safelySpreadBoxProps} from '@twilio-paste/box';
+import type {BadgeProps} from './types';
 
-import type {BadgeVariants, BadgeProps} from './types';
+export const isFocusableElement = ({as}: Partial<BadgeProps>): boolean => as !== 'span';
```

Comment:
Hm in case we add new variants here (i.e. div) can we prefer to be explicit and do `as === 'a' || as === 'button'` please?  The function name is `isFocusableElement` but this function expects knowledge of another component's internals. Best to make this function stand on its own for future proofyness

Reply:
Sure! Fixed.